pallets/msa/src/lib.rs

```diff
@@ -300,12 +300,6 @@ pub mod pallet {
 		/// The number of key values has reached its maximum
 		KeyLimitExceeded,
 
-		/// More than one account key exists for the MSA during retire attempt
-		MoreThanOneKeyExists,
-
-		/// Can't retire a registered provider MSA
-		RegisteredProviderCannotBeRetired,
```

Comment:
This error is now part of the CheckFreeExtrinsic use ValidityErrors.

Reply:
do we need to tag people from Sas team for any change that might break their end or do we have a process for this?